bowtie/tests/test_integration.py

```diff
@@ -2995,6 +3031,447 @@ async def test_suite_not_a_suite_directory(tmp_path):
+        tmp_path / "two.json",
+    )
+    buggy_v2_2019_stdout, _ = await bowtie(
+        "validate",
+        "-i",
+        miniatures.has_bugs_by_versions + ",version=2.0",
+        "-D",
+        "2019",
+        "--expect",
+        "valid",
+        tmp_path / "schema.json",
+        tmp_path / "one.json",
+        tmp_path / "two.json",
+    )
+
+    tar_path = tmp_path / "versioned_reports.tar"
+    tar_from_versioned_reports(
+        tar_path=tar_path,
+        id="buggy",
+        versions=frozenset(["1.0", "2.0"]),
+        versioned_reports=[
+            (
+                "1.0",
+                Dialect.by_short_name()["draft2020-12"],
+                buggy_v1_2020_stdout,
+            ),
+            (
+                "1.0",
+                Dialect.by_short_name()["draft2019-09"],
+                buggy_v1_2019_stdout,
+            ),
+            (
+                "2.0",
+                Dialect.by_alias()["2020"],
+                buggy_v2_2020_stdout,
+            ),
+            (
+                "2.0",
+                Dialect.by_alias()["2019"],
+                buggy_v2_2019_stdout,
+            ),
+        ],
+    )
+
+    jsonout, stderr = await bowtie(
+        "trend",
+        "-i",
+        "buggy",
+        tar_path,
+        "--format",
+        "json",
+        json=True,
+    )
+
+    assert (await command_validator("trend")).validated(jsonout) == [
+        [
+            "buggy",
+            [
+                [
+                    "draft2020-12",
+                    [
+                        [
+                            "2.0",
+                            {
+                                "failed": 0,
+                                "errored": 0,
+                                "skipped": 0,
+                            },
+                        ],
+                        [
+                            "1.0",
+                            {
+                                "failed": 2,
+                                "errored": 0,
+                                "skipped": 0,
+                            },
+                        ],
+                    ],
+                ],
+                [
+                    "draft2019-09",
+                    [
+                        [
+                            "2.0",
+                            {
+                                "failed": 2,
+                                "errored": 0,
+                                "skipped": 0,
+                            },
+                        ],
+                        [
+                            "1.0",
+                            {
+                                "failed": 0,
+                                "errored": 0,
+                                "skipped": 0,
+                            },
+                        ],
+                    ],
+                ],
+            ],
+        ],
+    ]
+    assert stderr == ""
+
+
+@pytest.mark.asyncio
+@pytest.mark.containers
```

Comment:
This marks this test as requiring containers. I don't see anything here requiring containers, you're using a direct connectable (which you should use as often as possible).